File: Vectors , ArrayList, List/Java/SumOfElements.java
// Find sum of all elements in ArrayList.
import java.util.ArrayList;

public class SumOfElements {
    public static void main(String[] args) {
        ArrayList<Integer> v = new ArrayList<>();
        v.add(2);
        v.add(4);
        v.add(6);
        v.add(8);
        v.add(10);

        int sum = 0;
        for(int val : v) {
            sum += val;
        }
        System.out.println("Sum of elements in ArrayList: " + sum);
    }
}

File: Vectors , ArrayList, List/C/SumOfElements.c
// Find sum of all elements in an array.
#include <stdio.h>

int main() {
    int v[] = {2, 4, 6, 8, 10};
    int sum = 0;
    int size = sizeof(v) / sizeof(v[0]);
    for(int i = 0; i < size; i++) {
        sum += v[i];
    }
    printf("Sum of elements in array: %d\n", sum);
    return 0;
}